pkg/scheduler/framework/plugins/clusteraffinity/cluster_affinity.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	cluster "github.com/karmada-io/karmada/pkg/apis/cluster/v1alpha1"
 	"github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
+	work "github.com/karmada-io/karmada/pkg/apis/work/v1alpha1"
```

Comment:
```suggestion
	clusterv1alpha1 "github.com/karmada-io/karmada/pkg/apis/cluster/v1alpha1"
	policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
	workv1alpha1 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha1"
```
Could you please help update the legacy import alias as well?